test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/metrics/MetricsUtils1Spec.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofilesrouter.connectors.metrics
+
+import com.codahale.metrics.{Counter, Histogram, MetricRegistry, Timer}
+import org.mockito.ArgumentMatchers.{anyString, endsWith}
+import org.mockito.Mockito
+import org.mockito.Mockito.{times, verifyNoInteractions, verifyNoMoreInteractions, when}
+import org.scalatest.compatible.Assertion
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class MetricsUtils1Spec extends AsyncFlatSpec {
```

Comment:
We should  find a better name for this class

Reply:
this need to be deleted. Forgot to do that

Reply:
done